tools/RAiDER/downloadGNSSDelays.py

```diff
@@ -19,16 +19,33 @@
 # base URL for UNR repository
 _UNR_URL = "http://geodesy.unr.edu/"
 
-def parse_args():
+def createParser():
```

Comment:
BTW the standard in python is to use `snake_case` for function and variable names.